src/package/arch.sh

```diff
@@ -9,21 +9,17 @@ PKG="pacman"
 puts BLUE "Using $PKG $ARCH"
 
 puts RED "update: starting"
-$PWR $PKG -Syy
+try $PWR $PKG -Syy
 puts GREEN "update: completed"
 
 puts RED "upgrade: starting"
-$PWR $PKG -Syu
+try $PWR $PKG -Syu
 puts GREEN "upgrade: completed"
 
-puts RED "clean all: starting"
-$PWR $PKG -R $($PKG -Qtdq)
-puts GREEN "clean all: completed"
```

Comment:
why these 3 lines are ❌ ?

Reply:
I don't know why that command was there, I can't even execute it and I discovered it only now by doing this commit...

Reply:
Got it, garbage leaved there for a while